icubam/www/handlers/db.py

```diff
@@ -83,3 +88,36 @@ def get(self, collection):
       os.remove(tmp_path)
     else:
       self.write(data.to_html())
+
+  @tornado.web.authenticated
```

Comment:
Same question about the permission model here. Can any authenticated user post data?!

Reply:
So normally only people with STATS UPLOAD or ALL tokens can post. @olivierteboul can you confirm?  We should also make the POST endpoint only accessible to ALL and UPLOAD keys (but STATS keys are a restricted set of people so also trusting here).

Reply:
This is a APIKeyProtectedHandler which means that the authentication is based on the API key. The key needs to exist, to be valid (not expired and active) and the restrictions must be one of   
 `store.AccessTypes.STATS, store.AccessTypes.ALL, store.AccessTypes.UPLOAD`